docs/docs/api-reference/valkey.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Valkey"
+---
+
```

Comment:
Please add an example. The docs generator won't generate cli commands without it. Additionally, when it is possible, please use it as a test template.

Reply:
Added example in https://github.com/aiven/aiven-operator/pull/865/commits/a7c52704af564d97713bd6be8ed6f7d4c9909a60